app/src/main/java/com/termux/app/TermuxActivity.java

```diff
@@ -367,7 +367,11 @@ public void onSessionFinished(final TerminalSession finishedSession) {
                         showToast(toToastTitle(finishedSession) + " - exited", true);
                 }
 
-                if (mTermService.getSessions().size() > 1) {
+                if (getPackageManager().hasSystemFeature(PackageManager.FEATURE_LEANBACK)) {
```

Comment:
Could you add a comment describing why this check is made (that the failsafe launcher icon doesn't show on Android TV)?

Reply:
There problem with additional launcher icons on Android TV devices. Tested on Android virtual device from SDK. I don't know if it will work for real devices but since on AVD it is not working, I guess the optimal choice will be left old behaviour there.

Reply:
Comments added.